components/web/php/php-7_0/Makefile

```diff
@@ -110,7 +110,7 @@ CONFIGURE_OPTIONS     +=	--with-pgsql=shared,$(PG_BINDIR)
 
 # Enabled Extensions
 #CONFIGURE_OPTIONS     +=	--with-libxml-dir=$(ECPREFIX)
-#CONFIGURE_OPTIONS     +=	--with-freetype-dir=$(ECPREFIX)
+CONFIGURE_OPTIONS     +=	--with-freetype-dir=$(ECPREFIX)
```

Comment:
1) ECPREFIX is not defined, it's a legacy derived from ec-userland, please, fix it with correct value. 
2) Please, bump COMPONENT_REVISION

Reply:
? I noticed ECPREFIX as an odd variable but interesting that it worked. I replaced it with /usr directly as with the jpeg dir done by the previous contributor. Is that Correct? Or would you like another prefix Variable